README.md

```diff
@@ -67,6 +67,48 @@ Complete the following steps before you deploy the Secure Landing Zone module.
 
     Select **All Identity and Access enabled services** when you assign access to the group.
 
+### Steps to prevent Destroy and recreation of cluster.
+1. Prior to `terraform plan` and `terraform apply`. Edit `moved_config.tf`
```

Comment:
Before jumping into the steps, users will need some brief explanation of what these steps accomplish for them. Something like,

`You can update to a new version of the Red Hat OpenShift pattern without destroying your cluster.`